climada_petals/hazard/low_flow.py

```diff
@@ -40,7 +40,6 @@
 from climada.hazard.base import Hazard
 from climada.hazard.centroids import Centroids
 import climada.util.coordinates as u_coord
-from climada.util.tag import Tag
```

Comment:
How did this work before? I don't see a `climada/util/tag.py` module in any of the recent releases

Reply:
after release 3.3, hazard.tag and entity.tag have been merged